cli/fuzz_main.cpp

```diff
@@ -29,7 +30,9 @@ class membuf : public std::basic_streambuf<char>
 class memstream : public std::istream
 {
 public:
-    memstream(const uint8_t *p, size_t l) : std::istream(&_buffer), _buffer(p, l)
+    memstream(const uint8_t *p, size_t l) :
+        std::istream(&_buffer),
+        _buffer(p, l)
```

Comment:
not from this change, but maybe better with the Dive naming convention of member name? (m_buffer)